src/releaseManager.ts

```diff
@@ -1,58 +1,57 @@
 import * as express from 'express';
-import * as cookieParser from 'cookie-parser';
 import * as httpProxy from 'http-proxy';
-import { URL } from 'url';
+import * as cookieParser from 'cookie-parser';
+import { noop } from 'lodash';
 
-import { health } from './health';
-import { environmentsByUrl } from './routingMap';
-import { createEnvNameGenerator } from './getEnvName';
-import { weightsByEnvironment } from './environments';
+import { health, setIsHealthy } from './health';
+import { redirectToHttps } from './redirectToHttps';
+import { getEnvForBranchPreview } from './branchPreviewer/getEnvForBranchPreview';
+import { getEnvForTrafficSplitting } from './trafficSplitter/getEnvForTrafficSplitting';
 
-const getEnvName = createEnvNameGenerator(weightsByEnvironment);
+process.on('unhandledRejection', () => {
+  setIsHealthy(false);
```

Comment:
We'll eventually want to log these somewhere, like SumoLogic?

Reply:
@msafih I'm thinking we should extract the log wrapper to a separate package so we can use it in all applications and take advantage of the typed events.